tests/test_core/test_profile.py

```diff
@@ -262,6 +262,25 @@ def test_entropy(self):
         entropy = got.entropy()
         assert_allclose(entropy, [2, 1])
 
+    def test_relative_entropy_terms(self):
+        """Check that relative_entropy_terms works for different background distributions"""
+        data = [[0.25, 0.25, 0.25, 0.25], [0.5, 0.5, 0, 0]]
+        got = MotifFreqsArray(array(data), "ABCD")
+        rel_entropy = got.relative_entropy_terms(background=None)
+        expected = [[0, 0, 0, 0], [-0.25, -0.25, -0.5, -0.5]]
+        assert_allclose(rel_entropy, expected)
+        background = {"A": 0.5, "B": 0.25, "C": 0.125, "D": 0.125}
+        rel_entropy = got.relative_entropy_terms(background=background)
+        expected = [[0.5, 0, -0.125, -0.125], [0, -0.25, -0.375, -0.375]]
+        assert_allclose(rel_entropy, expected)
+
```

Comment:
add a test for provision of a background which are not probabilities, i.e. values do not sum to one, and a test for when the values are < 0.

Reply:
You have not done this yet. You test that an assertion is raised as follows:
```python
# negative values not allowed
with self.assertRaises(ValueError):
    got.relative_entropy_terms(background=dict(A=-0.5, B=1.5))
# must sum to 1
with self.assertRaises(ValueError):
    got.relative_entropy_terms(background={"A": 0.5, "B": 0.25, "C": 0.125})
```